Assets/Scripts/TileManager.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TileManager : MonoBehaviour
+{
+    
+    public GameObject tilePrefab;
+    public Material pathTileMaterial;
+    public Material nonPathTileMaterial;
+    public Tile[,] tiles;
+
+
+    //level 1 for right now. Stars represent the path
+    //and the - represent open sapces
+    private char[,] level1 = { 
+        {'*','-','-','-','-','*','*','*','-','-'},
+        {'*','-','-','-','-','*','-','*','-','-'},
+        {'*','-','-','-','-','*','-','*','-','-'},
+        {'*','-','-','-','-','*','-','*','-','-'},
+        {'*','*','*','*','*','*','-','*','-','-'},
+        {'-','-','-','-','-','-','-','*','-','-'},
+        {'-','*','*','*','*','*','*','*','-','-'},
+        {'-','*','-','-','-','-','-','-','-','-'},
+        {'-','*','-','-','-','-','-','-','-','-'},
+        {'-','*','-','-','-','-','-','-','-','-'},};
+
+
+    // Start is called before the first frame update
```

Comment:
Please change to a summary block for consistency's sake, also please describe what the method does instead of when it's called. (Yes, I know this is the default comment from Unity but it's good to replace it once the method is being used)